src/Drips.sol

```diff
@@ -288,11 +288,13 @@ abstract contract Drips {
             receivableCycles = toCycle - fromCycle - maxCycles;
             toCycle -= receivableCycles;
         }
-        DripsState storage state = _dripsStorage().states[assetId][userId];
+        mapping(uint32 => AmtDelta) storage amtDeltas =
+            _dripsStorage().states[assetId][userId].amtDeltas;
         for (uint32 cycle = fromCycle; cycle < toCycle; cycle++) {
-            amtPerCycle += state.amtDeltas[cycle].thisCycle;
+            AmtDelta memory amtDelta = amtDeltas[cycle];
```

Comment:
This makes sense

Reply:
It actually saves a `sload`, I was a little surprised that it's not optimized better. Not all tips from C4 actually make a difference when benched, but this one does.